setup.cfg

```diff
@@ -34,6 +34,7 @@ universal = 1
 [flake8]
 max-line-length = 99
 exclude = docs, .git, __pycache__, .ipynb_checkpoints
+extend-ignore = DUO103 # insecure use of "pickle" or "cPickle"
```

Comment:
why is this error being raised and can we resolve it in the code?

Reply:
I believe we want to ignore this error in general. @csala mentioned it [here](https://github.com/sdv-dev/SDMetrics/pull/96#discussion_r743669901).

Reply:
That is correct @fealho
We do not intend to stop using `pickle` for the load/save functionality, so we can only ignore this.